src/components/header/header.scss

```diff
@@ -1,17 +1,25 @@
 .header-nav {
-  text-align: center;
-  padding-top: 20px;
   position: fixed;
-  left: 35vw;
+  padding-top: 20px;
+  left: calc( 50% - 90px); 
 
-  #logo {
+  
+  #logo {   
+    width: 200px;
+    height: auto;
     margin: 0 auto;
-    width: 300px;
+    float: none;
     
     @media only screen and (min-width: 700px) {
       width: 150px;
       position: fixed;
       left: 45vw;
     }
+
+    @media only screen and (min-width: 1000px) {
+      width: 150px;
+      position: fixed;
+      left: 45vw;
+    }
```

Comment:
No need to reiterate these as it's already been applied from the smaller screens. Only put stuff you intend to override.